release_utils/changelog.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import json
+import requests
+from subprocess import check_output
+from collections import OrderedDict
+
+
+class ChangeLogIndex(object):
+    """
+    Extracts a list of changelog entries to a text file
+    """
+
+    def cmd(self, c):
+        return check_output(c.split())
+
+    def __init__(self, path):
+        self.path = os.path.abspath(path)
+        conf_file = os.path.join(os.path.dirname(__file__), '../bl.conf')
```

Comment:
__file__ not recommended.
C'est pas une donnée consistante dans tous les cas, je pense qu'il vaudrait mieux que ca soit une argument du binaire et passé au __init__

Reply:
done